develop/plone/i18n/contribute_to_translations.rst

```diff
@@ -10,27 +10,17 @@ How to contribute to Plone core translations
 Introduction
 ============
 
-You need to have write access to
-https://github.com/collective/plone.app.locales to be able to commit
-your translation directly.
+You need to have write access to https://github.com/collective/plone.app.locales to be able to commit your translation directly.
```

Comment:
First, this is, in fact, write access to the collective. Which is explained here https://collective.github.io/

Also, this is not true. 
One may fork the repository and create a PR as explained below.